### PR TITLE
Add optional, new apps, some flatpak, and electrom directories

### PR DESCRIPTION
**Pull request by RlndVt**

### CODE REVIEW

File: rsync-homedir-excludes.txt [hunk 1]
```diff
@@ -47,6 +48,8 @@
 #
 #/games
 #.local/share/Steam
+#.var/app/com.valvesoftware.Steam/.local/share/Steam
+#.var/app/com.valvesoftware.Steam/.cache
```
Comment (rubo77) [inline rsync-homedir-excludes.txt]: This last line is obsolete

**Review by rubo77 — CHANGES_REQUESTED** [review]

**Comment by rubo77** [comment]
Maybe we should simply add

    .var/app/*/Cache

Instead?

**Comment by RlndVt** [comment]
> Maybe we should simply add
> 
> ```
> .var/app/*/Cache
> ```
> 
> Instead?

I've added lines for `.var/app/*/{,.}cache`, but (at least on my system) `Cache` does not hit any results:

`ls: cannot access '.var/app/*/Cache': No such file or directory`

**Comment by rubo77** [comment]
Maybe

    .var/app/**/Cache

**Comment by RlndVt** [comment]
As a blanket exclude for all Electron apps? Then we should also add:

```
.config/**/Application Cache
.config/**/blob_storage
.config/**/Cache
.config/**/CachedData
.config/**/Code Cache
.config/**/GPUCache
.var/app/**/Application Cache
.var/app/**/blob_storage
.var/app/**/Cache
.var/app/**/CachedData
.var/app/**/Code Cache
.var/app/**/GPUCache
```

**Comment by rubo77** [comment]
Yes please! ;)

File: rsync-homedir-excludes.txt [hunk 2]
```diff
@@ -93,6 +96,8 @@
 # Session-specific:
 /.dbus
 .cache
+.var/app/*/cache
```
Comment (rubo77) [inline rsync-homedir-excludes.txt:99]: This is obsolete now

Comment (RlndVt) [reply rsync-homedir-excludes.txt:99]: No it isn't, is it? `Cache` is not the same as `cache` (case sensitivity).

Comment (rubo77) [reply rsync-homedir-excludes.txt:99]: is there only one level between? or should it also need `**`? And does this not belong into the Electron Apps section too?

Comment (RlndVt) [reply rsync-homedir-excludes.txt:99]: These are the flatpak equivalent of `~/.cache`. Not necessarily a Electron app.

Comment (rubo77) [reply rsync-homedir-excludes.txt:99]: so Electron and flatpak both use `.var/app` ?

**Comment by RlndVt** [comment]
No, flatpak apps use `.var/app`. Electron apps can be installed as a normal
app (from your distro's repo), or as a flatpak

File: rsync-homedir-excludes.txt (hunk 2 above)
Comment (RlndVt) [reply rsync-homedir-excludes.txt:99]: No, flatpak apps use `.var/app`. Electron apps can be installed as a normal app (from your distro's repo), or as a flatpak

**Pull request merged**